webapp/src/main/java/com/google/sps/servlets/UploadServlet.java

```diff
@@ -76,13 +90,49 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             // Save the file on disk
             item.write(storeFile);
 
-            // TO DO: Call validator here 
+            // TO DO: get feed name from the user form
+            String feedName = "nl-openov";
+            NoticeContainer validatorNotices = runValidator(filePath, NUM_THREADS, feedName);
```

Comment:
As the number of threads is fixed, do we need to define this variable to the runValidator method?

Reply:
True, will remove this from the constructor.